src/elements/common/breadcrumbs/__tests__/InlineBreadcrumbs.test.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from 'react';
+import { IntlProvider } from 'react-intl';
+import { render, screen } from '@testing-library/react';
```

Comment:
if you use the render from the utils file (similar to `Breadcrumbs.test`), then you shouldn't need the `IntlProvider`